xcode/Subconscious/Shared/Components/Settings/GatewayURLSettingsView.swift

```diff
@@ -44,25 +44,54 @@ struct ResourceSyncBadge: View {
     }
 }
 
+struct RedeemInviteCodeLabel: View {
+    var status: ResourceStatus
+    
+    var label: String {
+        switch (status) {
+        case .initial:
+            return "Redeem Invite Code"
+        case .pending:
+            return "Redeeming..."
+        case .failed:
+            return "Failed to Redeem Invite Code"
+        case .succeeded:
+            return "Code Redeemed"
+        }
+    }
+    
+    var color: Color {
+        switch status {
+        case .failed:
+            return .red
+        default:
+            return .accentColor
+        }
+    }
+
+    var body: some View {
+        Label(title: {
```

Comment:
Minor nit: odd formatting. Prefer

```
Label(
    title: {
    },
    icon: {
    }
)
```

Reply:
Good catch, this was an Xcode autocomplete I didn't reformat.